Dockerfile

```diff
@@ -4,11 +4,14 @@ env PORT=8000
 workdir /app
 
 # Copy local version of the registry and install reqs
-copy requirements.txt /app
-run pip install -r requirements.txt
+copy pyproject.toml /app
 
 copy schemas /app/schemas
 copy marda_registry /app/marda_registry
+copy README.md /app/
```

Comment:
Should we be copying the `CONTRIBUTING.md` file, too? Not sure what the best practice is.

Reply:
yup, and LICENSE